### ISSUE TITLE
custom pretrained model - multiclass text classification issue

### ISSUE THREAD

**Issue by sbmark** [google-research/bert#695]
hi , I'm having a multiclass classification finetune problem.
input : one sentence 
output : 1800 number labels.  ex) [one, two, tree, four .... ]

I think it is a problem of learning rate, so I tried to test 2e-1 ~ 2e-6 / 5e-1~5e-6.
epoch : 3.0
max_seq_length : 100
train_batch_size : 32

but loss don't  converge. 

i change run_classifier.py 

    def get_examples(self, data_dir, ex_type):
        line_arr = []
        for root, dirs, files in os.walk(
                data_dir): 
            for fname in files:
                read_txt = os.path.join(root, fname)
                fr = open(read_txt, 'r')
                while True:
                    line = fr.readline()
                    if not line: break
                    strip_line = line.split('\t')
                    line_arr.append(strip_line)
                fr.close()
        if ex_type == "train":
            return self._create_examples(line_arr, "train")
        elif ex_type == "dev":
            return self._create_examples(line_arr, "dev")
        else:
            return self._create_examples(line_arr, "test")

    def get_labels(self, data_dir):
        dict = []
        read_txt = "./dict.txt"
        fr = open(read_txt, 'r')
        while True:
            line = fr.readline()
            if not line: break
            dict.append(line.strip())
        return dict
    def _create_examples(self, df, set_type):
        examples = []
        for (i, row) in enumerate(df):
            guid = "%s-%s" % (set_type, i)
            text_a = tokenization.convert_to_unicode(row[2].strip())
            labels = tokenization.convert_to_unicode((row[1].split('%'))[1].strip())
            examples.append( InputExample(guid=guid, text_a=text_a, text_b=None, label=labels))

        return examples

i have test_data, train_data, dev_data folder and data format = (xxx)\t(label)\t(sentence)\t
label data format : (label)\n(label)\n...

so result is : 

INFO:tensorflow:{'intent': 428, 'probabilities': array([ 0.00080097,  0.00015798,  0.00036487, ...,  0.00014206,
        0.00018639,  0.00012763], dtype=float32)}
INFO:tensorflow:[ 0.00080097  0.00015798  0.00036487  0.00044985  0.00048236  0.00072977
  0.00045677  0.00071928  0.0017894   0.00022954]
INFO:tensorflow:1847

INFO:tensorflow:{'intent': 428, 'probabilities': array([ 0.00080162,  0.00015639,  0.00036667, ...,  0.00014249,
        0.00018661,  0.00012805], dtype=float32)}
INFO:tensorflow:[ 0.00080162  0.00015639  0.00036667  0.00045948  0.00048472  0.00075246
  0.00045833  0.00072498  0.00166848  0.00023078]
INFO:tensorflow:1847

INFO:tensorflow:{'intent': 428, 'probabilities': array([ 0.00078834,  0.00015773,  0.00036384, ...,  0.00013955,
        0.0001838 ,  0.00012494], dtype=float32)}
INFO:tensorflow:[ 0.00078834  0.00015773  0.00036384  0.00044616  0.00047903  0.00072271
  0.00045926  0.000712    0.00163276  0.00022725]
INFO:tensorflow:1847
  
INFO:tensorflow:{'intent': 428, 'probabilities': array([ 0.00079147,  0.00016372,  0.00036187, ...,  0.00013362,
        0.00017847,  0.00012108], dtype=float32)}
INFO:tensorflow:[ 0.00079147  0.00016372  0.00036187  0.00043071  0.00047354  0.00070194
  0.00046688  0.00071655  0.00166785  0.00022914]
INFO:tensorflow:1847

INFO:tensorflow:{'intent': 428, 'probabilities': array([ 0.00074385,  0.00015681,  0.00035067, ...,  0.0001334 ,
        0.00017416,  0.00011894], dtype=float32)}
INFO:tensorflow:[ 0.00074385  0.00015681  0.00035067  0.00040286  0.00044398  0.00064551
  0.00043283  0.00068721  0.00169529  0.00021355]
INFO:tensorflow:1847

INFO:tensorflow:{'intent': 428, 'probabilities': array([ 0.00091541,  0.00017719,  0.00039133, ...,  0.00012684,
        0.00016818,  0.00011849], dtype=float32)}
INFO:tensorflow:[ 0.00091541  0.00017719  0.00039133  0.00045481  0.00048052  0.00073943
  0.00051825  0.00077433  0.00168681  0.00024464]

plz, I wait for an answer
